doc/source/tutorial/linalg.rst

```diff
@@ -353,10 +353,8 @@ where :math:`\mathbf{A}^{\dagger}` is called the pseudo-inverse of
 
 The command :obj:`linalg.lstsq` will solve the linear least-squares
 problem for :math:`\mathbf{c}` given :math:`\mathbf{A}` and
-:math:`\mathbf{y}` . In addition, :obj:`linalg.pinv` or
-:obj:`linalg.pinv2` (uses a different method based on singular value
-decomposition) will find :math:`\mathbf{A}^{\dagger}` given
-:math:`\mathbf{A}.`
+:math:`\mathbf{y}` . In addition, :obj:`linalg.pinv`will find
```

Comment:
I think this is due to the space
```suggestion
:math:`\mathbf{y}` . In addition, :obj:`linalg.pinv` will find
```

Reply:
Amazing how much I can screw up just by touching a file 🤦

Reply:
It's Monday it's normal 😉